src/js/gi/components/AccordionItem.jsx

```diff
@@ -18,7 +18,7 @@ class AccordionItem extends React.Component {
     const expanded = this.state.expanded;
     return (
       <li>
-        <button onClick={this.toggle()} className="usa-accordion-button" aria-expanded={expanded} aria-controls="amendment-1">
+        <button onClick={this.toggle} className="usa-accordion-button" aria-expanded={expanded} aria-controls="amendment-1">
```

Comment:
Not sure what best practice is around this, but this looks almost like inline javascript. Is it intended? Wouldn't it be better to do some kind of live binding for a button having this class name?